src/main/scala/io/appthreat/atom/slicing/UsageSlicing.scala

```diff
@@ -246,7 +246,17 @@ object UsageSlicing:
                         m.columnNumber.map(_.intValue())
                       )
                   )
-                  .l,
+                  .l ++ call.argument.inCall.map(c =>
```

Comment:
@buchi-busireddy This append operation results in new entries in the usages slices that captures the code and the types suitable for languages like python